Project_2/djangoProject/cars/files/cars/api_detail.py

```diff
@@ -0,0 +1,33 @@
+import requests
+account = {
+    "email": "test.test@victorysquarepartners.com",
+    "password": "Y4WUzqYFu3Dif8K",
+    'content-type': 'application/json'
```

Comment:
Just to be clear, the header does not need to be in the `.env` file and can be kept here hardcoded.

However, if you are only using this header inside the `api_detail` function, it would be better if it were moved there.